src/app.js

```diff
@@ -1,21 +1,47 @@
 'use strict';
+// - You can find a files to test this task in ./public folder.
+// |-- Its already tested with .html\.txt\.css files.
+// |-- Try link like this: http://localhost:/file/index.html =)
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const PORT = process.env.PORT || 8000;
+
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+// Server init:
+const server = http.createServer((request, response) => {
+  // Get file name from url params:
+  const fileFromParam = request.url.split('/').slice(1);
```

Comment:
```suggestion
  const [baseRoute, ...fileRoute] = request.url.split('/').slice(1);
```
Can be destructurized